instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2BedrockRuntimeTest.java

```diff
@@ -1417,4 +1423,478 @@ void testConverseStreamOptions() throws InterruptedException, ExecutionException
                             KeyValue.of("index", Value.of(0)),
                             KeyValue.of("content", Value.of("This model")))));
   }
+
+  @Test
+  void testInvokeModelAmazonTitan() {
+    BedrockRuntimeClientBuilder builder = BedrockRuntimeClient.builder();
+    builder.overrideConfiguration(createOverrideConfigurationBuilder().build());
+    configureClient(builder);
+    BedrockRuntimeClient client = builder.build();
+
+    String modelId = "amazon.titan-text-lite-v1";
+
+    Document requestPayload =
+        Document.mapBuilder()
+            .putString("inputText", "Say this is a test")
+            .putDocument(
+                "textGenerationConfig",
+                Document.mapBuilder()
+                    .putNumber("maxTokenCount", 10)
+                    .putNumber("temperature", 0.8f)
+                    .putNumber("topP", 1)
+                    .putList("stopSequences", singletonList(Document.fromString("|")))
+                    .build())
+            .build();
+
+    SdkJsonGenerator generator = new SdkJsonGenerator(new JsonFactory(), "application/json");
+    DocumentTypeJsonMarshaller marshaller = new DocumentTypeJsonMarshaller(generator);
+    requestPayload.accept(marshaller);
+
+    InvokeModelRequest request =
+        InvokeModelRequest.builder()
+            .modelId(modelId)
+            .body(SdkBytes.fromByteArray(generator.getBytes()))
+            .build();
+
+    InvokeModelResponse response = client.invokeModel(request);
+
+    JsonNode node = JsonNode.parser().parse(response.body().asByteArray());
+    Document responsePayload = node.visit(new DocumentUnmarshaller());
+    Document result = responsePayload.asMap().get("results").asList().get(0);
+    assertThat(result.asMap().get("outputText").asString())
+        .isEqualTo("\nHello! I am a computer program designed to");
+
+    getTesting()
+        .waitAndAssertTraces(
+            trace ->
+                trace.hasSpansSatisfyingExactly(
+                    span ->
+                        span.hasName("text_completion amazon.titan-text-lite-v1")
+                            .hasKind(SpanKind.CLIENT)
+                            .hasAttributesSatisfying(
```

Comment:
any reason for not using `hasAttributesSatisfyingExactly`?

Reply:
Yeah that was my feeling - for reference, these are the others

```
rpc.system,
    http.response.status_code,
    rpc.service,
    server.port,
    url.full,
    rpc.method,
    http.request.method,
    aws.request_id,
    server.address
```

These are populated by the "base" AWS SDK instrumentation and not the bedrock specific part. Notably, I felt the HTTP ones are a bit much. But what do you think? I don't mind going through all the tests to add them if it seems worth it.